emoji-copy@felipeftn/handlers/sql.js

```diff
@@ -43,6 +43,12 @@ export class SQLite {
     `);
   }
 
+  increment_selection(unicode) {
+    this.query(
+      `UPDATE emojis SET clicked_times = clicked_times + 1 WHERE unicode = '${unicode}'`
+    )
+  }
```

Comment:
Just little details to make the linter happy hahaha :smile: 
```suggestion
  increment_selection(unicode) {
    return this.query(`
      UPDATE emojis SET clicked_times = clicked_times + 1 WHERE unicode = '${unicode}'
    `);
  }
```